sample/legacy.go

```diff
@@ -0,0 +1,126 @@
+package sample
+
+import (
+	"sync"
+	"sync/atomic"
+
+	metrics "github.com/rcrowley/go-metrics"
+	rand "github.com/remerge/go-xorshift"
+)
+
+type LegacyLockFreeSample struct {
```

Comment:
Keeping the old implementation open makes no sense. All samples anyway will be replaced with new combined ones.